RELEASING.md

```diff
@@ -0,0 +1,82 @@
+# Releasing OpenTelemetry Packages (for maintainers only)
+This document explains how to publish all OT modules at version x.y.z. Ensure that you’re following semver when choosing a version number.
+
+Release Process:
+* [Create a new branch](#create-a-new-branch)
+* [Open a Pull Request](#open-a-pull-request)
+* [Create a Release](#Create-a-Release)
+* [Move stable tag](#Move-stable-tag)
+* [Update master](#Update-master)
+* [Check PyPI](#Check-PyPI)
+* [Troubleshooting](#troubleshooting)
+
+
+## Create a new branch
+The following script does the following:
+- update master locally
+- creates a new release branch `release/<version>`
+- updates version and changelog files
+- commits the change to a new branch `release/<version>-auto`
+
+*NOTE: This script was run by a GitHub Action but required the Action bot to be excluded from the CLA check, which it currently is not.*
```

Comment:
should we fiile an issue around this work? since it is a requirement for complete automation.

Reply:
There's an issue here: https://github.com/open-telemetry/community/issues/306 that tracks a migration to EasyCLA which *should* allow us to start using the GH actions bot again.